Source/Blazorise/Components/Check/Check.razor.cs

```diff
@@ -86,12 +86,12 @@ protected override void BuildClasses( ClassBuilder builder )
     /// Sets the check group name.
     /// </summary>
     [Parameter]
-    public string Group
+    public string Name
```

Comment:
This can be regular properly without the need for DirtyClass since we don't need to regenerate class names.

`public string Name { get; set; }`